### ISSUE TITLE
Significant performance differences in Total War: Troy between Heroic and Epic Games Store/Steam-Native

### ISSUE THREAD

**Issue by ms178** [Heroic-Games-Launcher/HeroicGamesLauncher#2767]
### Describe the bug

Total War: Troy comes with three in-game-benchmarks in the graphics option menu. I've observed a significant performance penalty in the second of these in-game-benchmarks from 133 fps > 78 fps when running the game via the Heroic Games Launcher vs. Epic Games Store/Steam-Native. The Proton version and command line options I've used are set to be the same for both launchers. 

`RADV_PERFTEST=sam,bolist RADV_DEBUG=shadowregs DXVK_ASYNC=1 %command%`

As the performance is just as expected in the other two benchmarks, I really wonder what's going wrong with that specific benchmark scene and why I only get to see this with Heroic.

### Add logs

```text
(14:43:03) INFO:    [Legendary]:        Refreshing library...
(14:43:03) INFO:    [Legendary]:        Refreshing Epic Games...
(14:43:03) INFO:    [Connection]:       Connectivity: check-online
(14:43:03) INFO:    [Connection]:       Pinging external endpoints
(14:43:03) INFO:    [Legendary]:        Running command: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(14:43:03) INFO:    [Legendary]:        Legendary location: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(14:43:03) INFO:    [Gog]:              GOGDL location: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(14:43:03) INFO:    [Gog]:              GOGDL location: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(14:43:03) INFO:    [Legendary]:        Game list updated, got 48 games & DLCs
(14:43:03) WARNING: [Backend]:          Protocol already registered.
(14:43:03) INFO:    [Gog]:              Running command: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/marcus/.config/heroic/gog_store/auth.json --version
(14:43:04) INFO:    [Connection]:       Connectivity: online
(14:43:04) INFO:    [Backend]:          Loading Screen Ready
(14:43:04) INFO:    [Backend]:          

Heroic Version: 2.8.0 Nico Robin
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.2

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: cachyos KERNEL: 6.3.5-4.1-cachyos-lto ARCH: x64
CPU: Intel Xeon® E5-2696 v3 @2.3 GOVERNOR: performance
RAM: Total: 62.64 GiB Available: 59.26 GiB
GRAPHICS: GPU0: Vega 10 XL/XT [Radeon RX Vega 56/64] VRAM: 512MB 
PROTOCOL: x11

(14:43:04) INFO:    [Frontend]:         Refreshing undefined Library
(14:43:04) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(14:43:04) INFO:    [Gog]:              Found 0 game(s) to update
(14:43:04) INFO:    [Backend]:          Frontend Ready
(14:43:04) INFO:    [Legendary]:        Running command: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(14:43:04) INFO:    [Backend]:          Checking for current version changelog
(14:43:04) INFO:    [Frontend]:         Updated to a new version, cleaaning up the cache.
(14:43:04) INFO:    [Legendary]:        Running command: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/legendary cleanup
(14:43:04) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(14:43:04) INFO:    [Legendary]:        Checking for game updates: /opt/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(14:43:05) INFO:    [Legendary]:        Found 0 games to update
(14:43:05) INFO:    [Frontend]:         No cache found, getting data from legendary...
(14:43:05) INFO:    [Legendary]:        Refreshing library...
(14:43:05) INFO:    [Frontend]:         Force Update
(14:43:06) INFO:    [Winetricks]:       Downloading Winetricks
(14:43:09) INFO:    [Backend]:          Starting the Download Queue
```

### Steps to reproduce

Run the second in-game-benchmark found via the graphics options menu.

### Expected behavior

I expect to get the same 133 fps, not 78 fps.

### Heroic Version

Latest Stable

### System Information

```
  Host: klx99 Kernel: 6.3.5-4.1-cachyos-lto arch: x86_64 bits: 64
    Desktop: KDE Plasma v: 5.27.5 Distro: CachyOS
Machine:
  Type: Desktop System: LENOVO product: GAMING TF v: N/A
    serial: <superuser required>
  Mobo: Lenovo model: X99-TF Gaming v: G368J V1.1
    serial: <superuser required> UEFI: American Megatrends v: CX99DE26
    date: 10/10/2020
CPU:
  Info: 18-core model: Intel Xeon E5-2696 v3 bits: 64 type: MT MCP cache:
    L2: 4.5 MiB
Graphics:
  Device-1: AMD Vega 10 XL/XT [Radeon RX 56/64] driver: amdgpu v: kernel
  Display: x11 server: X.Org v: 21.1.99 with: Xwayland v: 23.1.1 driver: X:
    loaded: amdgpu unloaded: modesetting dri: radeonsi gpu: amdgpu
    resolution: 2560x1440
  API: OpenGL v: 4.6 Mesa 23.2.0-devel (git-5a85c4432c) renderer: AMD
    Radeon RX Vega (vega10 LLVM 15.0.7 DRM 3.52 6.3.5-4.1-cachyos-lto)

```

**Comment by imLinguin** [comment]
I think the configuration is ultimately not the same. How do you run the Epic Games Store? Are you sure that Proton is running in runtime in both cases? Details you provided don't show full configuration.

**Comment by ms178** [comment]
I've imported EGS as a third-party game in Steam and I've made sure to use the same proton-cachyos in the compatibility tab and also use the same RADV options there in the starting options. In EGS, I've also edited the settings for the game to use the RADV options (but cannot tell if they are really applied).

Just a theory: Under the assumption that EGS doesn't apply the RADV options properly, I could imagine that `RADV_PERFTEST=sam` could cause such a large regression. I'll need to test that though.
